snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -1053,27 +1052,27 @@ public static void GroupByEx3()
         static class GroupBy4
         {
             // <Snippet125>
-            class Pet
+            class Pet(string name, float age)
             {
-                public string Name { get; set; }
-                public double Age { get; set; }
+                public string Name { get; set; } = name;
+                public float Age { get; set; } = age;

```

Comment:
This feels meaningless. Double is the default floating point type. I don't see a reason to specify `float`.